UndertaleModLib/Models/UndertaleGeneralInfo.cs

```diff
@@ -478,7 +510,33 @@ private long GetInfoNumber(long firstRandom, bool infoTimestampOffset)
     /// <inheritdoc />
     public override string ToString()
     {
-        return DisplayName + " (GMS " + Major + "." + Minor + "." + Release + "." + Build + ", bytecode " + BytecodeVersion + ")";
+        if (Major == 1)
+            return DisplayName + " (GMS " + Major + "." + Minor + "." + Release + "." + Build + ", bytecode " + BytecodeVersion + ")";
+        else
+        {
+            StringBuilder sb = new(DisplayName?.ToString() ?? "");
+            sb.Append(" (GMS ");
```

Comment:
```suggestion
            if (Major < 2022)
                sb.Append(" (GMS ");
            else
                sb.Append(" (GM ");
```

Reply:
If it's "GM**S** 2022.**2**" but "GM 2022.**3**", then:
```suggestion
            if (Major == 2022 && Minor >= 3)
                sb.Append(" (GM ");
            else
                sb.Append(" (GMS ");
```

Reply:
Ah, but then you would have to add `|| Major > 2022`. 2023.1 is already out.

I'll commit my version, if you please.

Reply:
Oh, yeah, I forgot.
But there's still no need for `||` - just make it `if (Major >= 2022 && Minor >= 3)`

Reply:
Oh